### ISSUE TITLE
In the task of cross-domain few-shot object detection, the category setting for the source domain and the target domain

### ISSUE THREAD

**Issue by zhilingdong** [lovelyqian/CDFSOD-benchmark#33]
Hello, thank you very much for your outstanding work. 
I would like to ask whether the new class categories in the target domain can overlap with the base class categories in the source domain in the task of cross-domain few-shot object detection.

**Comment by lovelyqian** [comment]
Yes, under the setting of CD-FSOD, there is no overlap between source and target classes.  But if you just want to use this method to improve your results, it is still okay to try this method.

**Comment by zhilingdong** [comment]
OK, thank you very much.
@lovelyqian
